components/HTMLContent.js

```diff
@@ -35,8 +52,29 @@ export const isEmptyValue = value => {
  * ⚠️ Be careful! This component will pass content to `dangerouslySetInnerHTML` so
  * always ensure `content` is properly sanitized!
  */
-const HTMLContent = styled(({ content, ...props }) => {
-  return content ? <div dangerouslySetInnerHTML={{ __html: content }} {...props} /> : <div {...props} />;
+const HTMLContent = styled(({ content, collapsable, sanitize, ...props }) => {
+  const [isOpen, setOpen] = React.useState(false);
```

Comment:
When possible, using the `Collapse` component is a good way to implement these behaviors as it relies on the standard `<details>` tag (making it accessible per default).

Reply:
Unfortunately not applicable here due to design requests, it could be useful if Collapse was more abstract than it currently is.